source/local/local.go

```diff
@@ -144,17 +149,37 @@ func (s *localSource) GetFeatures() *nfdv1alpha1.Features {
 	return s.features
 }
 
+func parseComments(lines [][]byte, fileName string) map[string]interface{} {
+	comments := make(map[string]interface{})
+	for _, l := range lines {
+		line := strings.TrimSpace(string(l))
+		if len(line) > 0 && strings.HasPrefix(line, "#") {
+			lineSplit := strings.SplitN(line, "=", 2)
+
+			key := lineSplit[0]
+			value := lineSplit[1]
+			switch key {
```

Comment:
Make this more generic to check if the comment starts with a `+`, in that case extract the key (and value)